js/ssai_pulse.js

```diff
@@ -270,6 +281,7 @@ OO.Ads.manager(function(_, $)
       if (amcAd && amcAd.ad)
       {
         window.open(amcAd.ad.clickthrough);
+        _handleTrackingUrls(amcAd, ["linearClickTracking"]);
```

Comment:
We should probably handle the tracking URL first in case the clickthrough pauses/exits the current video page and the user never comes back to the video page. We wouldn't be able to monetize that click in this instance.